CONTRIBUTING.md

```diff
@@ -2,15 +2,18 @@
 
 We love pull requests. Here's a quick guide.
 
+_Please refer to the README.md for information about the structure of this repo (note that this repo supports our packages `datadog-cdk-constructs` and `datadog-cdk-constructs-v2`_)
+
 1. Fork, clone and branch off `main`:
     ```bash
     git clone git@github.com:<your-username>/datadog-cdk-constructs.git
     git checkout -b <my-branch>
     ```
-2. Install the repositories dependencies, `yarn install`.
+2. Install the repositories dependencies, `yarn install`. Do this within the repo root, as well as within the project version you plan to edit (`v1` or `v2`).
 3. Make your changes.
-4. Test your changes using CDK commands such as `cdk synth`, `cdk diff`, and `cdk deploy`
-5. Ensure the unit tests pass:
+4. Navigate to the root of the `v1` or `v2` directory
+4. Manually test your changes using CDK commands such as `cdk synth`, `cdk diff`, and `cdk deploy`.
+5. Ensure the unit tests pass (test command must also be run in `v1` or `v2` root):
```

Comment:
These numbers look a bit off (3, 4, 4, 5). I think Markdown will correct them automatically though. I actually think the easiest thing is to just number everything as `1.` and let Markdown do the work. That way it's easy to add and remove stuff in the future without making the code look weird.

Reply:
Great idea, i changed everything to be `1.`